### PR TITLE
ELPP-3693 Run on dynamic Jenkins node

### PR DESCRIPTION
**Pull request by giorgiosironi**
Rather than on `containers--medium` which is manually managed

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -6,8 +6,7 @@ elifePipeline {
         commit = elifeGitRevision()
     }
 
-    elifeOnNode(
-        {
+    node('containers-jenkins-plugin') {
             stage 'Build images', {
```
Comment (thewilkybarkid) [inline Jenkinsfile]: Indentation could be reduced?

**Comment by giorgiosironi** [comment]
Now green with the latest AMI (which ships coveralls credentials for everything) (but only `profiles` has some). I am looking for a way to streamline the AMI update, but that's a separate matter.

File: Jenkinsfile (hunk 1 above)
Comment (giorgiosironi) [reply Jenkinsfile]: Yes, I didn't want to pollute the diff too much

Comment (thewilkybarkid) [reply Jenkinsfile]: That’s what `?w=1` is for. :)

**Pull request merged**